frontend/src/components/App/TopBar.tsx

```diff
@@ -179,6 +193,7 @@ export function PureTopBar({
   const { t } = useTranslation('frequent');
   const isSmall = useMediaQuery('(max-width:960px)');
   const isMedium = useMediaQuery('(max-width:960px)');
+  const dispatch = useDispatch();
```

Comment:
To keep this separated from redux, a `versionOnClick` could be passed in to the PureTopBar.

Reply:
I used dispatch because we have two places where we can trigger the dialog from, the cluster chooser + the user menu.
So this is a global way of triggering the build dialog.